### ISSUE TITLE
Throws an exception if `err.stack` is undefined

### ISSUE THREAD

**Issue by simonexmachina** [floatdrop/express-error-with-sources#3]
If a middleware calls `next(new Error('Foobar'))` then this module throws an exception. You should probable check if `err.stack` is defined before calling `parsetrace(err)`. 

Here's my workaround FYI:

```
    fn = require('express-error-with-sources') contextSize: 3
    app.use (err, req, res, next)->
      if err.stack
        fn err, req, res, next
      else
        next err
```

**Comment by floatdrop** [comment]
This is fixed in 1.3.0.

**Comment by simonexmachina** [comment]
Great, thanks. Turns out npm update express-error-with-sources wasn’t
upgrading me to 1.3.0.
